contrib/xfburn/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "xfburn"
+pkgver = "0.7.0"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+    "xfce4-dev-tools",
+]
+makedepends = [
+    "exo-devel",
+    "glib-devel",
+    "gst-plugins-base-devel",
+    "gstreamer-devel",
+    "gtk+3-devel",
+    "libburn-devel",
+    "libgudev-devel",
+    "libisofs-devel",
+    "libxfce4ui-devel",
+]
+depends = ["gst-plugins-bad", "gst-plugins-base", "gst-plugins-good"]
```

Comment:
i don't think the explicit plugin dependencies are needed?

Reply:
not strictly, no